code/modules/surgery/external_repair.dm

```diff
@@ -0,0 +1,215 @@
+//Procedures in this file: Organic limb repair
+//////////////////////////////////////////////////////////////////
+//						LIMB REPAIR SURGERY						//
+//////////////////////////////////////////////////////////////////
+/datum/surgery_step/repairflesh/
+	priority = 1
+	can_infect = 1
+	blood_level = 1
+	req_open = 1
+
+/datum/surgery_step/repairflesh/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	if (target.stat == DEAD) // Sorry defibs, your subjects need to have pumping fluids for these to work.
+		return 0
+	if (isslime(target))
+		return 0
+	if (target_zone == O_EYES || target_zone == O_MOUTH)
+		return 0
+	if (!hasorgans(target))
+		return 0
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	if (affected == null)
+		return 0
+	if (affected.is_stump())
+		return 0
+	if (affected.robotic >= ORGAN_ROBOT)
+		return 0
+	return 1
+
+
+//////////////////////////////////////////////////////////////////
+//						SCAN STEP								//
+//////////////////////////////////////////////////////////////////
+
+/datum/surgery_step/repairflesh/scan_injury
+	allowed_tools = list(
+	/obj/item/weapon/autopsy_scanner = 100,
+	/obj/item/device/healthanalyzer = 50,
```

Comment:
Bump this up to 80, please. Not everyone who's gonna want to do this surgery (ie. antags) is going to have an autopsy scanner at hand.